### PR TITLE
chore: use davinci github actions directly

### PR DESCRIPTION
**Pull request by dmaklygin**
### Description

We need to use davinci GH Actions directly

### How to test

- Run CI command

### Development checks

- [x] Add changeset according to [guidelines](https://github.com/toptal/picasso/blob/master/docs/contribution/changeset-guidelines.md) (if needed)
- [x] Read [CONTRIBUTING.md](https://github.com/toptal/picasso/blob/master/CONTRIBUTING.md) and [Component API principles](https://github.com/toptal/picasso/blob/master/docs/contribution/component-api.md)
- [x] Ensure that all PR checks are green
- [x] Self reviewed

<details>
<summary>PR commands</summary>
<br />

List of available commands:

- `@toptal-bot run all` - Run whole pipeline
- `@toptal-bot run build` - Check build
- `@toptal-bot run deploy:documentation` - Deploy documentation
- `@toptal-bot run package:alpha-release` - Release alpha version

</details>

<details>
<summary>PR Review Guidelines</summary>
<br />

#### When to approve? ✅

**You are OK** with merging this PR and

1. You have no extra requests.
2. You have optional requests.
   1. Add `nit:` to your comment. (ex. `nit: I'd rename this variable from makeCircle to getCircle`)

#### When to request changes? ❌

**You are not OK** with merging this PR because

1. Something is broken after the changes.
2. Acceptance criteria is not reached.
3. Code is dirty.

#### When to comment (neither ✅ nor ❌)

**You want your comments to be addressed** before merging this PR in cases like:

1. There are leftovers like unnecessary logs, comments, etc.
2. You have an opinionated comment regarding the code that requires a discussion.
3. You have questions.

#### How to handle the comments?

1. An owner of a comment is the only one who can resolve it.
2. An owner of a comment must resolve it when it's addressed.
3. A PR owner must reply with ✅ when a comment is addressed.

</details>

### CODE REVIEW

**Review by TomasSlama — APPROVED** [review]
I'm just wondering when we upgrade from v2 to v3, there is probably some breaking change, so it does not affect actions that we use here?

**Review by OndrejTuma — APPROVED** [review]
I love it 🙂

**Review by ozgurkececioglu — APPROVED** [review]
💪

File: .github/dependabot.yml [hunk 1]
```diff
@@ -11,3 +11,15 @@ updates:
   labels:
     - no-jira
   versioning-strategy: increase
+- package-ecosystem: github-actions
+  directory: "/"
+  schedule:
+    interval: weekly
+    time: "07:00"
+  open-pull-requests-limit: 4
```
Comment (dmaklygin) [inline .github/dependabot.yml:19]: I'll decrease to 2

**Comment by OndrejTuma** [comment]
@toptal-bot run package:alpha-release

**Review by denieler — APPROVED** [review]

File: .github/dependabot.yml [hunk 2]
```diff
@@ -11,3 +11,15 @@ updates:
   labels:
     - no-jira
   versioning-strategy: increase
+- package-ecosystem: github-actions
+  directory: "/"
+  schedule:
+    interval: weekly
+    time: "07:00"
+  open-pull-requests-limit: 2
```
Comment (OleksandrNechai) [inline .github/dependabot.yml:19]: why `2`? :)

**Review by OleksandrNechai — APPROVED** [review]

File: .github/dependabot.yml (hunk 2 above)
Comment (dmaklygin) [reply .github/dependabot.yml:19]: we've already had 2 PRs per week for NPM packages. It used to be 4, @denieler asked to decrease. I agree :)

**Pull request merged**